lahja/asyncio/endpoint.py

```diff
@@ -735,22 +617,36 @@ def stop(self) -> None:
             return
 
         # Broadcast to every connected Endpoint that is allowed to receive the event
+        if remotes_for_broadcast is None:
```

Comment:
nitpick: This might be cleaner as:

```python
if remotes_for_broadcast is None:
  remotes_for_broadcast = (
    remote for remote in self._connections
    if remote.can_send_item(item, config)
  )

await self._send_to_remotes(item, config, *remotes_for_broadcast)
```